src/load_games.py

```diff
@@ -21,38 +64,47 @@ def load_games(path: str = 'resources/PGN_database',
             path to games database folder.
             Default: resources/PGN_database
 
-        is_rated : bool
-            filter rated games.
-            {True -> only rated,
-             False -> only casual,
-             None -> all games}
-            Default: None
-
     Returns
     -------
         List[dict]
             list of games in DB, sorted by date.
 
     """
 
+    fname = []
+    if os.path.isdir(path):
+        fname = flatten_list(
+            list(
+                map(lambda x:
+                    list(map(lambda y:
+                             os.path.join(x[0], y), x[2])),
+                    os.walk(path)))
+        )
+
+    elif os.path.exists(path):
+        fname = [path]
+
     games = list()
-    for file in os.listdir(path):
-        with open(os.path.join(path,
-                               file), 'r') as f:
-            ind_game = ast.literal_eval(f.read())
-            games.append(ind_game)
+    for file in fname:
```

Comment:
`file` is already used by Python.

```suggestion
    for filename in filenames:
```